build.gradle

```diff
@@ -119,8 +119,14 @@ ext {
   addParametersForTests = { name, options ->
     // -parameters generates arguments with parameter names in TestInfo#getDisplayName.
     // ref: https://github.com/junit-team/junit5/blob/4c0dddad1b96d4a20e92a2cd583954643ac56ac0/junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTest.java#L161-L164
-    if (name == "compileTestJava" || name == "compileTestScala")
+    if (name == "compileTestJava" || name == "compileTestScala") {
       options.compilerArgs << "-parameters"
+    } else if (name == "compileJava") {
```

Comment:
please change `compileJava` to `compileScala` to add the args to "scala compiler"